layouts/shortcodes/full-changelog.html

```diff
@@ -187,6 +240,14 @@ <h3>{{- .product -}}</h3>
       }
     }
   }
+  const productAreaFilter = urlParams.get("product-area");
+  if (productAreaFilter) {
+    for (const productArea of productAreaDropdown) {
+      if (productArea.value === productAreaFilter.toLowerCase()) {
+        productAreaDropdown.value = productAreaFilter.toLowerCase();
```

Comment:
```suggestion
        productAreaDropdown.value = productAreaFilter.toLowerCase();
        break;
```
No need to continue iterating the `for` loop.